velox/functions/prestosql/aggregates/SetAggregates.cpp

```diff
@@ -274,9 +287,32 @@ class SetAggAggregate : public SetBaseAggregate<T> {
 
     auto tracker = Base::trackRowSize(group);
     rows.applyToSelected([&](vector_size_t i) {
+      if (throwOnNestedNulls_) {
+        checkNulls(Base::decoded_, i);
+      }
+
       accumulator->addValue(Base::decoded_, i, Base::allocator_);
     });
   }
+
+ private:
+  bool checkNulls(const DecodedVector& decoded, vector_size_t index) const {
```

Comment:
return value is not used; change to void

Reply:
Fixed